pkg/standalone/standalone.go

```diff
@@ -85,8 +86,11 @@ const (
 	healthPort = 58080
 	metricPort = 59090
 
-	schedulerHealthPort = 58181
-	schedulerMetricPort = 59191
+	schedulerHealthPort = 58081
+	schedulerMetricPort = 59091
+	schedulerEtcdPort   = 52379
```

Comment:
do we want to set both the grpc & http ports for etcd? Otherwise it throws some warning in etcd logs about it not being prod ready if only 1 is set

Reply:
I saw that. I don't think it is a problem for standalone mode. I can go either way here.